qualtran/bloqs/block_encoding/chebyshev_polynomial_test.py

```diff
@@ -221,6 +221,7 @@ def test_chebyshev_matrix():
 
```

Comment:
TestBlockEncoding in this file is still passing QAny rather than a tuple of registers